pages/common/nasm.md

```diff
@@ -0,0 +1,23 @@
+# nasm
+
+> The Netwide Assembler, a portable 80x86 assembler.
+
+- Assemble raw binary file `source` from `source.asm`:
+
+`nasm {{source.asm}}`
+
+- Assemble binary file `output_file` of the specified format:
```

Comment:
It's confusing to use "raw binary file" in the first example and just "binary file" in the second, since this suggests that they might be different. I'd prefer if we used the same description in both cases.